.github/no-response.yml

```diff
@@ -1,7 +1,7 @@
 # Configuration for probot-no-response - https://github.com/probot/no-response
 
 # Number of days of inactivity before an Issue is closed for lack of response
-daysUntilClose: 10
+daysUntilClose: 15
```

Comment:
`no-response` will wait for 15 days before closing anything.

I'm still not sure if we must wait that long. Anyway, I don't think it's time to change it. We need time to see if this setting will be useful.